CovertInfiltration/Src/CovertInfiltration/Classes/UIListener_CovertActionCompleted.uc

```diff
@@ -0,0 +1,55 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  statusNone
+//  PURPOSE: This class is used to track when a covert action is completed in order to
+//	apply soldier will loss
+//---------------------------------------------------------------------------------------
+//  WOTCStrategyOverhaul Team
+//---------------------------------------------------------------------------------------
+
+class UIListener_CovertActionCompleted extends UIScreenListener config(Infiltration);
+
+var config float MIN_WILL_LOSS;
+var config float MAX_WILL_LOSS;
```

Comment:
Please leave a comment that it's percent/multiplier (+ in config). Right now it's kinda hard to understand what it is expressed in - eg. i would assume absolute units (eg. 22 will points)